packages/admin-ui/src/filters/filter/filter.tsx

```diff
@@ -26,32 +26,35 @@ export function Filter<T>(props: FilterProps<T>) {
     combobox.selectedItem && getOptionId(combobox.selectedItem)
 
   return (
-    <BaseFilter
-      state={state}
-      appliedValuesLabel={
-        <SingleItemLabel appliedItem={appliedItem} state={state} />
-      }
-    >
-      {items.map((item) => {
-        const itemId = getOptionId(item)
-
-        return (
-          <ComboboxItem
-            aria-selected={itemId === currentSelectedId}
-            key={itemId}
-            value={itemId}
-            focusOnHover
-            hideOnClick={false}
-            onClick={() => combobox.setSelectedItem(item)}
-            style={style.option}
-            id={`${baseId ?? ''}-item-${itemId}`}
-          >
-            <FilterRadio checked={itemId === currentSelectedId} />
-            {getOptionLabel(item)}
-          </ComboboxItem>
-        )
-      })}
-    </BaseFilter>
+    <>
```

Comment:
similar to the [previous case](https://github.com/vtex/admin-ui/pull/252/files#r898432463)?